README.md

```diff
@@ -101,6 +101,8 @@ These builds are maintained by the community. While they should be safe, downloa
 
 * Windows Package Manager (winget): [Usage](https://docs.microsoft.com/en-us/windows/package-manager/winget/)
 
+* Homebrew Formulae (Mac only):[Download](https://formulae.brew.sh/cask/freetube)
```

Comment:
Mac only part is incorrect

Reply:
There's only a Mac build being provided so it is Mac only (even though homebrew can be used on Linux).

(source: https://formulae.brew.sh/api/cask/freetube.json )
```json
{
"token":"freetube",
...
"url":"https://github.com/FreeTubeApp/FreeTube/releases/download/v0.19.1-beta/freetube-0.19.1-mac-x64.dmg",
...
}
```